doc/Learn/Tutorials/Markup/HowTo-CreateMarkupProject.md

```diff
@@ -0,0 +1,487 @@
+---
+uid: Learn.Tutorials.HowToCreateMarkupProject
+---
+
+# How to set up your own C# Markup project
+
+In this tutorial you will learn how to set up a new *basic project* using Uno Platform and C# Markup and using some basic resources and also make a comparison between C# Markup and XAML.
+
+The purpose of this section of the tutorial is to show you how to create a basic project from scratch.
+
+## Set up a Markup project
+
+You can use this tutorial to learn how to set up a Uno Platform project.
+We will use the comparison between two projects, one being C# Markup and the second XAML.
+
+> The tutorials below can teach you how to create both projects.
+
+- [Setting up the environment and creating the Markup project](xref:Learn.Tutorials.HowToMarkupProject)
+
+- [Setting up the environment and creating the XAML project](xref:Learn.Tutorials.HowToXamlProject)
+
+## Comparing the structures
+
+Open the two projects created and compare the structure of the two.
+
+### Comparing the C# Markup with the XAML project
+
+- Comparing projects structure
+
+    # [**C# Markup**](#tab/cs)
+
+    #### C# Markup project structure
+
+    - The project is separated into Solution Items, where general settings and properties files are located, and Source, which contains information about the Backend, Platforms and Shared Project.
+
+    - Check out how your first Start Up Markup project will look like.
+
+        ![Screenshot showing how to check your project's Markup initial structure using Uno Platform within Visual Studio.](../Assets/MarkupProject-InitialProject.jpg)
```

Comment:
The path looks incorrect

Reply:
it probably is... we're keeping the HowTo tutorials unlisted for now... they don't really belong in Markup but they are related.